tutorials/live_templates/template_support.md

```diff
@@ -3,70 +3,128 @@ title: 1. Adding Live Template Support
+#### Register Extension Points
+Using the `com.intellij.defaultLiveTemplates` and `com.intellij.liveTemplateContext` extension points, register the implementations with the IntelliJ Platform.
+The `file` attribute in the `defaultLiveTemplates` element only specifies `path/filename` under the `src/main/resources` folder, where the IntelliJ Platform expects to find it.
+In this example, having "liveTemplates" in the pathname is coincidental and has no particular significance.
+The filename does not include the file extension.
+```xml
+  <extensions defaultExtensionNs="com.intellij">
+    <defaultLiveTemplates file="/liveTemplates/Markdown"/>
+    <liveTemplateContext implementation="org.intellij.sdk.liveTemplates.MarkdownContext"/>
+  </extensions>
+```
+Now go to the [Check Plugin](#check-plugin) section to test the template.
+
+### IntelliJ Platform Versions v2019.3 and Earlier
+For older versions of the IntelliJ Platform follow this section to complete the implementation and register the extension points.
+Then proceed to the [Check Plugin](#check-plugin) section.
+ 
+#### Implement DefaultLiveTemplatesProvider
+The `MarkdownTemplateProvider` tells the IntelliJ Platform where to find the Live Template settings file. 
+Make sure to include the full path to the file, relative to the `src/main/resources` directory, excluding the file extension.
+```java
+package org.intellij.sdk.liveTemplates;
+
+import com.intellij.codeInsight.template.impl.DefaultLiveTemplatesProvider;
+import org.jetbrains.annotations.Nullable;
+
+public class MarkdownTemplateProvider implements DefaultLiveTemplatesProvider {
+  @Override
+  public String[] getDefaultLiveTemplateFiles() {
+    return new String[]{"liveTemplates/Markdown"};
+  }
+
+  @Nullable
+  @Override
+  public String[] getHiddenLiveTemplateFiles() {
+    return null;
+  }
+}
+```
+
+#### Register Extension Points
+Using the com.intellij.defaultLiveTemplatesProvider and com.intellij.liveTemplateContext extension points, register the implementations with the IntelliJ Platform.
 ```xml
   <extensions defaultExtensionNs="com.intellij">
     <defaultLiveTemplatesProvider implementation="org.intellij.sdk.liveTemplates.MarkdownTemplateProvider"/>
     <liveTemplateContext implementation="org.intellij.sdk.liveTemplates.MarkdownContext"/>
   </extensions>
 ```
 
+Now go to the [Check Plugin](#check-plugin) section to test the template.
+
 ## Check Plugin
-Now check that the plugin is working correctly. Run the plugin and verify there is a new entry under *File \| Settings \| Live Templates \| Markdown \| \[*. Finally, create a new file `Test.md` and confirm that the Live Template works.
\ No newline at end of file
+Now verify the plugin is working correctly. 
+Run the plugin in a Development Instance and verify there is a new entry under **File \| Settings \| Live Templates \| Markdown \| \{** (SDK: New link reference). 
```

Comment:
formatting looks strange

Reply:
Agreed. It's true to what appears, but looks odd in the text. I'll make the "(SDK: New link reference)" bold as well.